### ISSUE TITLE
How to use this ?

### ISSUE THREAD

**Issue by dh-saurabh** [ynqa/ktop#3]
It will be great if you can add some examples about how to use this library.

**Comment by ynqa** [comment]
@dh-saurabh Yeah, the current usages are not enough. I'll add them later. And also if you have some helpful examples, it's okay to submit them as PRs!
